src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/TypeOperations.cs

```diff
@@ -155,9 +155,10 @@ public override int Run(InterpretedFrame frame)
             {
                 if (frame.Peek() == null)
                 {
-                    frame.Pop();
-                    throw new InvalidOperationException();
+                    // Trigger InvalidOperationException with same localised method as if we'd called the Value getter.
```

Comment:
A case where the Americans are the traditionalists, ignoring the early 19th Century fad in Britain and Ireland that never died away. Agree it should be en-US, but old habits…